### ISSUE TITLE
USER ISSUE: Objects not showing up

### ISSUE THREAD

**Issue by Khalanix** [StrangeLoopGames/EcoIssues#7105]
**Version:** 0.7.3.2 beta

Not sure if the reason are too many objects in one area or it also might be the glass behind it. Right after I made a stockpile full of glass this bug started

**Expected behavior:**
Everything should load up and is expected to be able to interact with.

**Actual behavior:**
Objects are not loading into the world, sometimes relogging helps but the game isn't playable like that.

**Comment by Baithoven** [comment]
It happens only in this area 179,440 
Also in single player mode.


Cant upload world to git.
here is the world file:
[](https://drive.google.com/file/d/1xsRnk2EHkTTyU91Wu4PsSEEeYHUE5EMu/view?usp=sharing)

**Comment by Khalanix** [comment]
For reproduction:
If you relog, some objects load back into the world, but only if you relog near them.
If you walk away long enough and go back the objects won't load either.
In the mine below the house you can dig for stone but no chunks are spawning, only after a relog there are some parts of it on the ground.
Hope that helps to solve that bug or get a workaround

**Comment by MarjieAVolk** [comment]
Closing as duplicate of https://github.com/StrangeLoopGames/EcoIssues/issues/6693

Thanks for the details, and the save file.  I will paste your comments over there.
